packages/eslint-plugin-office-addins/test/rules/no-navigational-load.test.ts

```diff
@@ -0,0 +1,65 @@
+import { ESLintUtils } from '@typescript-eslint/experimental-utils'
+import rule from '../../src/rules/no-navigational-load';
+
+const ruleTester = new ESLintUtils.RuleTester({
+	parser: '@typescript-eslint/parser',
+});
+
+ruleTester.run('no-navigational-load', rule, {
```

Comment:
Need to include test with nested property references like range.format.font.load("color") from the spread sheet.  That one is a valid case, but something like range.format.load("font") would be invalid.  The spreadsheet also notes the use of range.load({format: { font: { color: true } } }) being a valid case.

Reply:
The code right is only checking for the last property and checking if the same of this property is navigational or scalar.
So like, it can detect that range. format.load("font") would be invalid if the font is characterized as a navigational property.

Then the case currently not supported is the range.load({format: { font: { color: true } } }), which I will work on

Reply:
Should have a test case for that nested property

Reply:
I created a similar one which is already checked in:
```
var range = context.workbook.getRange();
                range.load({borders: { fill: { color: true } } });
                if (myRange.borders.fill.color);
```
This one specifically (with format), would not pass as format is one of the 9 properties considered ambiguous.

Reply:
I'm talking about a range.borders.fill.load("color") . . . not the object format.

Reply:
Actually didn't work before.
I added the code to detect these cases now in the utils/load and also added some test cases.

The nice thing of using utils is that this bug also existed for other rules (they never worked on navigational) and now all of the rules using load will also accept all formats of load.